config/site_settings.yml

```diff
@@ -2372,6 +2372,13 @@ developer:
       - auto
       - enabled
     default: auto
+  glimmer_topic_list_mode:
```

Comment:
Should this be a "group" setting so that we can opt in and out by joining / leaving a group? 🤔

Reply:
Not yet. Until it supports customizations (and the themes and plugins use the new APIs) you wouldn't really want to use it in production. Then we might change it to be group-enabled, but by the time it does have customizations, we'll have an `auto` mode here that auto-enables it, so we might skip the group stage entirely.